### PR TITLE
[PP-40] Update heroku env variables in deploy process

### PR DESCRIPTION
**Pull request by andreikaptsiuh**
Task: https://andrei-lysiuk.atlassian.net/browse/PP-40
Description:
- add REACT_APP_SOCKET_URL and REACT_APP_BASE_URL variables
- add REACT_APP_BASE_URL to all requests or as base_url in http service
- check all requests should work
- update heroku envs on deploy stage (after deploy FE should connect to BE with correct link)

### CODE REVIEW

File: src/api/HttpClient.test.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ describe('Create class', () => {
 
 describe('Base url', () => {
   test('Should return url', () => {
-    expect(httpClient.http.defaults.baseURL === 'http://localhost:3000').toBe(true);
+    expect(httpClient.http.defaults.baseURL === process.env.REACT_APP_BASE_URL).toBe(true);
```
Comment (mrakofff) [inline src/api/HttpClient.test.ts]: remove this test

File: src/api/HttpClient.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export class HTTPClient {
     this._initializeInterceptors();
   }
 
-  static getInstance(baseURL = 'http://localhost:3000'): HTTPClient {
+  static getInstance(baseURL = process.env.REACT_APP_BASE_URL): HTTPClient {
```
Comment (mrakofff) [inline src/api/HttpClient.ts]: pass this as parameter but not default value. Default should be empty string

File: .env [hunk 3]
```diff
@@ -1 +1,2 @@
-REACT_APP_SOCKET_URL=ws://localhost:3000
\ No newline at end of file
+REACT_APP_SOCKET_URL=ws://localhost:3000
+REACT_APP_BASE_URL=http://localhost:3000/
```
Comment (mrakofff) [inline .env]: remove `/` at the end

**Review by mrakofff — APPROVED** [review]

File: .env (hunk 3 above)
Comment (andreikaptsiuh) [reply .env]: fix

File: src/api/HttpClient.ts (hunk 2 above)
Comment (andreikaptsiuh) [reply src/api/HttpClient.ts]: fix

File: src/api/HttpClient.test.ts (hunk 1 above)
Comment (andreikaptsiuh) [reply src/api/HttpClient.test.ts]: removed

**Review by MakFill — APPROVED** [review]

**Pull request merged**